### PR TITLE
Update to kmp 0.2.6

### PR DESCRIPTION
**Pull request by dangeross**

### CODE REVIEW

File: Android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,6 +2,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.CAMERA" />
```
Comment (ademar111190) [inline Android/app/src/main/AndroidManifest.xml]: Why camera permission?

Comment (dangeross) [reply Android/app/src/main/AndroidManifest.xml]: Good catch. I was copying the Blockstream Green permissions

Comment (ademar111190) [reply Android/app/src/main/AndroidManifest.xml]: Feel free to keep the internet permission; remove just the camera stuff

Comment (dangeross) [reply Android/app/src/main/AndroidManifest.xml]: Done

**Review by ademar111190 — APPROVED** [review]

**Pull request merged**